### ISSUE TITLE
Write custom loss function code to match Yang's loss function

### ISSUE THREAD

**Issue by mwinton** [mwinton/w266-final-project#39]

**Comment by chjatala** [comment]
"running average over the batches": what are the parameters for the running average?
- Is it simple moving average?  In that case, how many previous batches average is calculated? 
- Or is it cumulative moving average? In that case, is it computed from the start of training till current training step, or over a single epoch?
- Or is it exponential moving average? What is discount factor?

**Comment by chjatala** [comment]
> Confirmed that Keras is taking averages over batches:
> 
> "For training loss, keras does a running average over the batches. For validation loss, a conventional average over all the batches in validation data is performed. The training accuracy is the average of the accuracy values for each batch of training data during training."
> 
> [keras-team/keras#10426](https://github.com/keras-team/keras/issues/10426)

"running average over the batches": what are the parameters for the running average?

    Is it simple moving average? In that case, how many previous batches average is calculated?
    Or is it cumulative moving average? In that case, is it computed from the start of training till current training step, or over a single epoch?
    Or is it exponential moving average? What is discount factor?
